frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditOptionChoice/EditOptionList/EditOptionList.test.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import { EditOptionList } from './EditOptionList';
+import { screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
+import { ComponentType } from 'app-shared/types/ComponentType';
+import userEvent, { type UserEvent } from '@testing-library/user-event';
+import { componentMocks } from '../../../../../../../testing/componentMocks';
+import { renderWithProviders, optionListIdsMock } from '../../../../../../../testing/mocks';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+import { createQueryClientMock } from 'app-shared/mocks/queryClientMock';
+import type { FormComponent } from '../../../../../../../types/FormComponent';
+
+// Test data:
+const mockComponent: FormComponent<ComponentType.Dropdown> = componentMocks[ComponentType.Dropdown];
+const optionsIdMock = optionListIdsMock[0];
+mockComponent.optionsId = optionsIdMock;
+
+const handleComponentChangeMock = jest.fn();
+const getOptionListIds = jest
+  .fn()
+  .mockImplementation(() => Promise.resolve<string[]>(optionListIdsMock));
+
+describe('EditOptionList', () => {
+  it('should render the component', async () => {
+    renderEditOptionList();
+    expect(await screen.findByText(textMock('ux_editor.options.upload_title'))).toBeInTheDocument();
+  });
```

Comment:
This test case can be done without async:
```suggestion
  it('should render the component', () => {
    renderEditOptionList();
    expect(screen.getByText(textMock('ux_editor.options.upload_title'))).toBeInTheDocument();
  });
```